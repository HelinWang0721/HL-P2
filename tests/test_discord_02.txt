a, b;
main {
    public: b;
    private: a;
    nested {
        public: b;
        private: x;
        a = b;
        nested2 {
            nested3 {
                b = a;
            }
        }
        x = b;
        a = x;
    }
}
