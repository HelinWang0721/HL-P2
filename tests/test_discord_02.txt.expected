::a = nested.b
nested.b = ::a
nested.x = nested.b
::a = nested.x
